test/functional/page_objects/visual_builder_page.ts

```diff
@@ -67,13 +67,16 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
     }
 
     public async checkTimeSeriesChartIsPresent() {
-      await testSubjects.existOrFail('timeseriesChart');
+      const isPresent = await find.existsByCssSelector('.tvbVisTimeSeries');
+      if (!isPresent) {
+        throw new Error(`TimeSeries chart is not loaded`);
+      }
     }
 
     public async checkTimeSeriesLegendIsPresent() {
-      const isPresent = await find.existsByCssSelector('.tvbLegend');
+      const isPresent = await find.existsByCssSelector('.echLegend');
       if (!isPresent) {
-        throw new Error(`TimeSeries legend is not loaded`);
+        throw new Error(`TimeSeries chart is not loaded`);
```

Comment:
typo with `chart` since method name is `checkTimeSeriesLegendIsPresent`

Reply:
Fixed